spec/root-apps/systemjs.spec.js

```diff
@@ -80,6 +81,7 @@ describe("SystemJS loader :", () => {
 			happyUnload();
 			invalidUnload();
 			lifecyleProps();
+      registerWithObject();
```

Comment:
We've got some tabs vs spaces stuff here -- maybe we should follow this pr up with a pr that converts all the tabs to spaces?